windows/setup.ps1

```diff
@@ -77,29 +81,79 @@ foreach ($adapter in $ActiveNetcfg)
+Set-Attr $result.ansible_facts "ansible_bios_date" $win32_bios.ReleaseDate.ToString("MM/dd/yyyy")
+Set-Attr $result.ansible_facts "ansible_bios_version" $win32_bios.SMBIOSBIOSVersion
+Set-Attr $result.ansible_facts "ansible_hostname" $env:COMPUTERNAME
+Set-Attr $result.ansible_facts "ansible_fqdn" ($ip_props.Hostname + "." + $ip_props.DomainName)
+Set-Attr $result.ansible_facts "ansible_processor" $cpu_list
+Set-Attr $result.ansible_facts "ansible_processor_cores" $win32_cpu.NumberOfCores
+Set-Attr $result.ansible_facts "ansible_processor_count" $win32_cs.NumberOfProcessors
+Set-Attr $result.ansible_facts "ansible_processor_threads_per_core" ($win32_cpu.NumberOfLogicalProcessors / $win32_cs.NumberOfProcessors / $win32_cpu.NumberOfCores)
+Set-Attr $result.ansible_facts "ansible_processor_vcpus" ($win32_cpu.NumberOfLogicalProcessors / $win32_cs.NumberOfProcessors)
+Set-Attr $result.ansible_facts "ansible_product_name" $win32_cs.Model.Trim()
+Set-Attr $result.ansible_facts "ansible_product_serial" $win32_bios.SerialNumber
+Set-Attr $result.ansible_facts "ansible_product_version" ([string] $win32_cs.SystemFamily)
 Set-Attr $result.ansible_facts "ansible_system" $osversion.Platform.ToString()
+Set-Attr $result.ansible_facts "ansible_system_description" ([string] $win32_os.Description)
+Set-Attr $result.ansible_facts "ansible_system_vendor" $win32_cs.Manufacturer
 Set-Attr $result.ansible_facts "ansible_os_family" "Windows"
 Set-Attr $result.ansible_facts "ansible_os_name" ($win32_os.Name.Split('|')[0]).Trim()
 Set-Attr $result.ansible_facts "ansible_distribution" $osversion.VersionString
 Set-Attr $result.ansible_facts "ansible_distribution_version" $osversion.Version.ToString()
+Set-Attr $result.ansible_facts "ansible_distribution_major_version" $osversion.Version.Major.ToString()
+Set-Attr $result.ansible_facts "ansible_kernel" $osversion.Version.ToString()
+
+Set-Attr $result.ansible_facts "ansible_machine_id" $user.User.AccountDomainSid.Value
+Set-Attr $result.ansible_facts "ansible_domain" $ip_props.DomainName
+Set-Attr $result.ansible_facts "ansible_nodename" ($ip_props.HostName + "." + $ip_props.DomainName)
+Set-Attr $result.ansible_facts "ansible_windows_domain" $win32_cs.Domain
 
+Set-Attr $result.ansible_facts "ansible_owner_name" ([string] $win32_cs.PrimaryOwnerName)
+Set-Attr $result.ansible_facts "ansible_owner_contact" ([string] $win32_cs.PrimaryOwnerContact)
+
+Set-Attr $result.ansible_facts "ansible_user_dir" $env:userprofile
+Set-Attr $result.ansible_facts "ansible_user_gecos" ([string] $user.Label)
+Set-Attr $result.ansible_facts "ansible_user_id" $env:username
+Set-Attr $result.ansible_facts "ansible_user_uid" ([int] $user.User.Value.Substring(42))
+Set-Attr $result.ansible_facts "ansible_user_sid" $user.User.Value
+
+# Use English locale
+[System.Threading.Thread]::CurrentThread.CurrentCulture = [System.Globalization.CultureInfo]::GetCultureInfo('en-US')
```

Comment:
The problem was that on a Flemish Windows, the day-of-week was in Flemish. Your suggestion works fine. Expect a fix !